src/subarraynew.jl

```diff
@@ -0,0 +1,36 @@
+typealias NullableSubArray{T,N,P<:NullableArray,IV,LD} SubArray{T,N,P,IV,LD}
+
+@inline function Base.isnull(V::NullableSubArray, I::Real...)
+    @boundscheck checkbounds(V, I...)
+    @inbounds return V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
```

Comment:
What did you try, and what error did you get?  Don't splat `I`.  Just ` V.parent.isnull[Base.reindex(V, V.indexes, I)...]`.